### ISSUE TITLE
No semantic error displaying

### ISSUE THREAD

**Issue by StevenCarre** [microsoft/vscode#123440]
Issue Type: <b>Bug</b>

When I type syntax errors in php, they are not underlined in squiggly red lines anymore. 

VS Code version: Code 1.56.0 (cfa2e218100323074ac1948c885448fdf4de2a7f, 2021-05-04T21:58:14.757Z)
OS version: Linux x64 5.8.0-50-generic snap

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7400 CPU @ 3.00GHz (4 x 3362)|
|GPU Status|2d_canvas: unavailable_software<br>gpu_compositing: disabled_software<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: disabled_off<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: unavailable_software<br>webgl2: unavailable_software|
|Load (avg)|2, 2, 1|
|Memory (System)|7.68GB (1.62GB free)|
|Process Argv|--force-user-env --no-sandbox --unity-launch --no-sandbox --crash-reporter-id f4d73163-6880-4b81-922d-dbdabd8d90e5|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|x11|
</details><details><summary>Extensions (8)</summary>

Extension|Author (truncated)|Version
---|---|---
gitlens|eam|11.4.1
php-debug|fel|1.15.1
php-intellisense|fel|2.3.14
php-pack|fel|1.0.2
twig-language-2|mbl|0.9.2
php-namespace-resolver|Meh|1.1.8
git-graph|mhu|1.30.0
env-cmd-file-syntax|Nix|0.2.2

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
pythonvspyt602cf:30294773
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30291489
pythontb:30283811
vspre833:30267464
pythonptprofiler:30281270
vshan820:30294714
pythondataviewer:30285071
vscus158cf:30286554
bridgeflight:30299254

```

</details>

**Comment by JacksonKearl** [comment]
Please file an issue against the PHP extension you are using.

**Comment by StevenCarre** [comment]
Hello,


I have tried disabling all my extensions and the issue remains. It doesn't seem to be related to an extension.
Can you please reopen the issue?


Best regards
Steven Carré

Le 11 mai 2021 à 0:06, VS Code Issue Triage Bot ***@***.***> a écrit :






This issue is caused by an extension, please file it with the repository (or contact) the extension has linked in its overview in VS Code or the marketplace for VS Code. See also our issue reporting guidelines.

Happy Coding!

**Comment by JacksonKearl** [comment]
Yes, with all extensions disabled it is expected that you would not receive any error markings. But the issue is still with the extension, which once gave you error squiggles but now does not.
